src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -432,14 +430,7 @@ private static void checkPreconditions(Project project) {
         project.subprojects(subproject -> {
             subproject.afterEvaluate(sub -> {
                 sub.getPluginManager().withPlugin("nebula.dependency-recommender", plugin -> {
-                    RecommendationProviderContainer container =
-                            sub.getExtensions().findByType(RecommendationProviderContainer.class);
-                    if (container.getStrategy() == RecommendationStrategies.OverrideTransitives) {
-                        throw new GradleException("Must not use strategy OverrideTransitives for "
-                                + sub
-                                + ". "
-                                + "Use this instead: dependencyRecommendations { strategy ConflictResolved }");
-                    }
+                    NebulaUtils.verifyRecommendationStrategy(sub);
```

Comment:
this won't work if nebula is brought in via `plugins{}` i.e. in a different classloader.